src/translator_ingest/ingests/pathbank/pathbank_rig.yaml

```diff
@@ -0,0 +1,228 @@
+    terms_of_use_url: "https://pathbank.org/about"
+    license_name: "See PathBank About/Downloads for reuse and citation guidance"
+  data_access_locations:
+    - "All downloads: https://pathbank.org/downloads"
+  data_provision_mechanisms:
+    - file_download
+  data_formats:
+    - csv
+    - pwml
+  data_versioning_and_releases: "Treat each pull as a dated snapshot; downloads list 'Generated On' timestamps and large per-format archives."
+
+ingest_info:
+  ingest_categories:
+    - primary_knowledge_provider
+  utility: "Comprehensive pathway membership and mechanistic process data (reactions, transports, interactions) for Translator reasoning and pathway-centric analytics."
+  scope: "This ingest uses only two inputs: the master pathways CSV for biolink:Pathway nodes and the PWML archive for entities, processes, and edges. Other formats available from PathBank (BioPAX, SBML, SBGN, RXN, SVG, PNG, FASTA, SDF) are not currently used but may be considered for future enrichment."
+
+  relevant_files:
+    - file_name: "pathbank_all_pathways.csv.zip"
+      location: "https://pathbank.org/downloads"
+      description: "Master CSV with pathway subjects and descriptions; used to build Pathway nodes with IDs, names, descriptions, and species metadata."
+    - file_name: "pathbank_all_pwml.zip"
+      location: "https://pathbank.org/downloads"
+      description: "Per-pathway PWML files for detailed composition and connectivity (entities, processes, participants, localization, complexes)."
+
+  included_content:
+    - file_name: "pathbank_all_pathways.csv.zip"
+      included_records: "All pathway rows extracted to pathbank_pathways.csv"
+      fields_used: "pathway_id, pathway_name, description, species/taxon, external references if present"
+    - file_name: "pathbank_all_pwml.zip"
+      included_records: "All PWML pathway files"
+      fields_used: "entities (compounds, proteins, complexes, nucleic acids), processes (reactions, transports, interactions), participants (inputs/outputs), enzymes/transporters, localization (compartments/tissues)"
+
+  # Note: filtered_content is omitted because all records from the included files (pathbank_pathways.csv and PWML files) are used without filtering.
+
+  future_considerations:
+    - category: edge_property_content
+      consideration: "Edge properties such as stoichiometry and participant_role - may be added in future if needed and if supported by Biolink Model"
+    - category: edge_content
+      consideration: "Additional file formats available from PathBank (BioPAX, SBML, SBGN, RXN, SVG, PNG, FASTA, SDF) may be considered for future enrichment to provide additional pathway representations and visualizations"
+    - category: edge_content
+      consideration: "Additional interaction type mappings may be added as more PathBank interaction types are discovered and validated against Biolink Model predicates"
+    
+target_info:
+  infores_id: infores:translator-pathbank-kgx
+  edge_type_info:
```

Comment:
@mbrush - do you think part of the RIG (next iteration) could be selecting the subclass of Association that makes most sense for these parameters?  

It seems like a more specific association should be in Biolink for this data as it doesn't quite fit in:
molecular activity to pathway association
chemical to pathway association 
gene to pathway association
(one uses "part of" as its predicate, the other does not, but both seem like they are trying to describe the same sorts of edges as this is)...lots of other associations here that are close but not exactly the same as these big, bundling associations with lots of different categories for subjects and objects.  

if nothing else, we should put a ticket in for refining these down to more specific associations in the next round?